### DISCUSSION TITLE
2006/07/28/%E7%BE%8E%E5%8C%96%E7%A8%8B%E5%BA%8F-%E9%9A%90%E8%97%8F%E7%A8%8B%E5%BA%8F%E6%A1%86%E5%9B%BE%E4%B8%8A%E7%9A%84%E5%A4%A7%E4%B8%AA-cluster

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#100] [Announcements]
# 2006/07/28/%E7%BE%8E%E5%8C%96%E7%A8%8B%E5%BA%8F-%E9%9A%90%E8%97%8F%E7%A8%8B%E5%BA%8F%E6%A1%86%E5%9B%BE%E4%B8%8A%E7%9A%84%E5%A4%A7%E4%B8%AA-cluster

在编写某些程序的时候可能会遇到如图1 所示的情形：即用到了一个极为复杂的数据类型常量。 这个常量由于体积巨大，使得在程序框图无论怎么摆放都让人看起来不太舒服。如何才能把这个程序改造得美观一些呢？

https://blog.qizhen.xyz/2006/07/28/%E7%BE%8E%E5%8C%96%E7%A8%8B%E5%BA%8F-%E9%9A%90%E8%97%8F%E7%A8%8B%E5%BA%8F%E6%A1%86%E5%9B%BE%E4%B8%8A%E7%9A%84%E5%A4%A7%E4%B8%AA-cluster

**Comment by ruanqizhen** [comment]
之前博客下的留言：

刘伟印
2008年08月25日 14:42 编辑
不错，有意思，呵呵

回复
辉
2007年08月13日 00:24 编辑
谢谢大侠 

回复
捷周
2007年01月3日 13:45 编辑
不错，多谢分享！！！

回复
Qizhen
2006年07月31日 19:14 编辑
写 LabVIEW 程序一定要养成把代码整理干净漂亮的习惯。这样在每次构造一个新的 Cluster 类型，自然而然就会采用以上的方法，不会觉得有太多麻烦。这样却为以后的维护省去了几倍的时间。
关于 Type Def. 和 Strict Type Def. 过几天讨论 :)

回复
永青
2006年07月31日 09:29 编辑
这个方法我已经被阮大侠教育过了，还是不错的！

回复
Jing
2006年07月30日 10:43 编辑
好是挺好，就是麻烦了点。

回复
来来
2006年07月29日 00:18 编辑
路过，欢迎来访

回复
来来
2006年07月29日 00:18 编辑
路过，欢迎来访

**Comment by ruanqizhen** [comment]
之前博客下的留言：


捷周
2007年01月3日 13:55 编辑
楼主言之有理，不过个人觉得５年太久了．

回复
weili
2006年08月17日 23:03 编辑
这么多有意思的东西，很佩服你的坚持劲，居然把这么多东西写得这么透彻~

回复
竹贤
2006年08月17日 22:23 编辑
每个人的能力、特长、追求等都不同，所以不可能有一条适合所有人的标准的道路。关键是能否找到一个最适合自己的位子，这既有个人的能力的条件，更有机遇的限制。个人努力当然是必不可少的，但很多时候往往是可遇而不可求。
升职与成功不能划等号；丰厚的酬金是重要前提，但不是唯一的标准；关键是你个人的感觉和心态。职务的分布总是金字塔形的，在底层的总是最多数，如果人人升迁，成了倒金字塔形，能站得住吗？

回复
yoyo
2006年08月2日 19:12 编辑
你什么时候跳？
